service-api/src/main/java/greencity/dto/employee/SaveEmployeeDto.java

```diff
@@ -0,0 +1,37 @@
+package greencity.dto.employee;
+
+import greencity.annotations.ValidPhoneNumber;
+import greencity.dto.position.PositionDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.List;
+
+@Data
+@Builder
```

Comment:
Is it required to have setters and builders?

Reply:
done